src/mixer.cc

```diff
@@ -0,0 +1,195 @@
+#include "mixer.h"
+#include <sstream>
+
+
+namespace cycamore {
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::toolkit::ResBuf;
```

Comment:
SG: https://google.github.io/styleguide/cppguide.html#Namespaces "Do not use using-directives." and https://google.github.io/styleguide/cppguide.html#Aliases "However, local convenience aliases are fine in function definitions, private sections of classes, explicitly marked internal namespaces, and in .cc files:"  

I believe that inside of the function body where they are used is the safest and what the rest of the code base does.